ios/Staging-Info.plist

```diff
@@ -21,7 +21,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>47</string>
```

Comment:
Why change this to a hard-coded number?

Reply:
I haven't touched that field, this was done by Xcode internally. Actually I only modified two files directly: `Fastfile` and fastlane's README. All other modifications are the result of creating and configuring a new *scheme* and a new *target* in Xcode.

Reply:
I was wrong, this was done automatically by fastlane when a new build is pushed. I'm setting up all this again in a new branch to get a cleaner result.